upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -556,10 +556,10 @@ func (c *ApplyClusterCmd) Run() error {
 	}
 
 	bootstrapScriptBuilder := &model.BootstrapScript{
-		NodeUpConfigBuilder:        renderNodeUpConfig,
-		NodeUpSourceHash:           "",
-		NodeUpSource:               c.NodeUpSource,
-		AddAwsEnvironmentVariables: false,
+		NodeUpConfigBuilder:          renderNodeUpConfig,
+		NodeUpSourceHash:             "",
+		NodeUpSource:                 c.NodeUpSource,
+		AddMinioEnvironmentVariables: false,
```

Comment:
Can you please rename this to something slightly generic? Like- AddExtraEnvVriable? 

I shouldn't have started with AddAwsEnvironmentVariables.